src/cli/run_command.zig

```diff
@@ -192,6 +192,22 @@ pub const RunCommand = struct {
 
                     delimiter = 0;
                 },
+                // TODO: handle escape sequences properly
+                // https://github.com/oven-sh/bun/issues/53
+                '\\' => {
+                    delimiter = 0;
+
+                    switch (script[entry_i + 1]) {
```

Comment:
does this need a bounds check for entry_i + 1 < script.len

Reply:
`"foo": "bar\\\"`

Reply:
Oops &minus; while that's not valid JavaScript/JSON, we shouldn't segfault either... 😓